WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -316,13 +316,16 @@ import WordPressShared
                 // momentarily visible.
                 featuredImageView.image = nil
                 var url = featuredImageURL
+                let size = CGSize(width:featuredMediaView.frame.width, height:featuredMediaHeightConstraintConstant)
```

Comment:
Same issue as above.  Also, we don't need to preserve the original post content for the Reader like we do for a user's blog, so for the Reader it would be an optimization to rewrite URLs for private wpcom sites in `ReaderPostServiceRemote` and store the url rather than adjust the url at display time.   You might ping @astralbodies, I think he mentioned he might look into that a bit.